chanzuckerberg/single-cell-data-portal
# feat: Add gene migration incomplete banner (#241)

### Reviewers
**Functional:**
@tihuan 

**Readability:** 
@seve 

---

## Changes
- add `Banner` component that shows the incomplete gene migration banner on specific collections.
- modify collection component to show the banner to "data consumers".

## Definition of Done (from ticket)
Banner shown for specified collections.

## QA steps (optional)
Locally:
- Run locally and add local collection ID to `INCOMPLETE_GENE_MIGRATIONS_COLLECTIONS`.
- Go to the collection page on data portal. Banner should be displayed.

Rdev:
- https://ypark-rdev-frontend.rdev.single-cell.czi.technology/collections/50b7e0d4-ecc8-4576-a815-6529e02c883c
- https://ypark-rdev-frontend.rdev.single-cell.czi.technology/collections/423cdd2a-d096-4e87-a9a9-18e5168b45e7


<img width="1792" alt="Screen Shot 2021-10-05 at 13 45 58" src="https://user-images.githubusercontent.com/75650148/136077679-4e6dd8ab-f987-4ed6-a58c-6377df979d0d.png">

<img width="522" alt="Screen Shot 2021-10-05 at 13 46 14" src="https://user-images.githubusercontent.com/75650148/136077695-1df4c5d7-fe34-4e11-a553-3d9129196ea4.png">
